### PR TITLE
Broken Code

### PR DESCRIPTION
**Pull request by mithilc**
@twang Changed Stewie2 to make it incorrect.

### CODE REVIEW

**Comment by Pokemaster3733** [comment]
Well because you said you made Stewie incorrect I now you didn't mean this but, you forgot static in public static void code. I like how you made the last two lines into a method.

**Comment by Pokemaster3733** [comment]
You said that you changed Stewie2 in order to make it incorrect so I know you didn't mean to do this but, you didn't put static in public static void code. Otherwise I like how you made the last two lines into a method.

**Comment by twang** [comment]
Thanks @mithilc and @Pokemaster3733!

File: brokencode.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+public class brokencode {
+    public static void main(String args[]) {
+        System.output.("////////////////////");
+        code;
+       
+    }
+    public void code
```
Comment (twang) [inline brokencode.java]: Also don't forget the parenthesis after code
```
public static void code()
```

**Pull request closed without merging**